extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js

```diff
@@ -41,23 +41,55 @@ RdfSkeletonListDialog.prototype._constructBody = function(body) {
         '</p>').appendTo(body);
     
    var container = $('<div id="body" style="display:block;">');
-    var html = $('<div id="left-panel">');
+    var html = $('<div style="float:left;">');
     $.post("command/rdf-extension/list-rdf-skeletons",function(data){
        console.log(data);
-       var list = $('<select multiple width="300" style="width: 300px">');
-        var data = data['list'];
+       var list = $('<select multiple width="300" style="width: 300px;">');
+        var data = data.list
         for (var i = 0; i < data.length; i++){
-           $('<option id='+ i +' value="' + data[i] + '">' + data[i] + '</option>').appendTo(list);
-           $('option#' + i).click(function(evt){
+           var element = data[i];
+           var option = $('<option id='+ i + '>' + element.name + '</option>');
+           
+           option.click(function(evt){
                evt.preventDefault();
-               $("p.preview").text(evt.target.value);
+               $(".preview").html('');
+               console.log(data[event.target.id].skeleton);
+               var skeleton = JSON.parse(data[event.target.id].skeleton);
+               $("<h2>prefixes</h2>").appendTo(".preview")
+               var prefList = $("<ul>");
+               for (var x = 0; x < skeleton.prefixes.length; x++){
+                   $("<li>" + skeleton.prefixes[x].name + 
+                           " - <a href='" + skeleton.prefixes[x].uri + "'>"  
+                           + skeleton.prefixes[x].uri + 
+                           "</a></li>").appendTo(prefList);
+               }
+               $("</ul>").appendTo(prefList);
```

Comment:
A general comment on your jquery usage: you don't have to provide closing tags since these are DOM operations. See a speed comparison [here](https://stackoverflow.com/a/268520/4627267). Removes some clutter from the code as well, making it easier to read and even easier to maintain.